bindings/Sofa/tests/Core/BaseLink.py

```diff
@@ -97,7 +97,7 @@ def test_read(self):
         root = Sofa.Core.Node("root")
         c1 = root.addObject("MechanicalObject", name="t1")
         link = root.findLink("object")
-        self.assertEqual(link.getValueString(),"@/t1")
+        self.assertEqual(link.getValueString(),"@//t1", F"Link value should be '@//t1', but it is {link.getValueString()}")
```

Comment:
Really ?

Reply:
This does not seems to come from SofaPython3 side. `"@//t1"` is the value returned by Sofa...